tests/test_client.py

```diff
@@ -690,14 +691,15 @@ def test_token_refresh_failure(self):
         for status_code in REFRESH_STATUS_CODES:
             http = HttpMockSequence([
                 ({'status': status_code}, b''),
-                ({'status': '400'}, b'{"error":"access_denied"}'),
+                ({'status': http_client.BAD_REQUEST},
+                 b'{"error":"access_denied"}'),
             ])
             http = self.credentials.authorize(http)
             try:
                 http.request('http://example.com')
-                self.fail('should raise AccessTokenRefreshError exception')
-            except AccessTokenRefreshError:
-                pass
+                self.fail('should raise HttpAccessTokenRefreshError exception')
```

Comment:
While you're mucking with this code, mind switching from `self.fail` and catching the error to just using the modern approach: `self.assertRaises`?